packages/oracles-starter-kit/README.md

```diff
@@ -519,7 +519,7 @@ npx witnet-toolkit try-query --from-solidity ./contracts/witnet-requests/<contra
aJBzfR3u7dkOA24a9XL76aJWJvyNR1UmemU0+Lod6xcpwgawqrMqYUKBGvoVW0adct4rDjDP39IZfPoU2u95eUDuiFyaWehLk/ZEq9zrvlqVO1jl9Oo1W9UEHDXG1+bXPKVSaWtWFtVXU7e51823Cn6VqVD6VjjQk2/iI6kMbYR8VirR+ntZsmnbhrRirWgV2zVPhhLeK9jQxT1InSmAS+vfC0YdOpRAeOnHEjh/bXpBA/Fc1q+NhrQU6qqzHTKF9LIC1PieTLWctlrGN36eEEbpBm16dXKaF7OUreoI4DaA44Bwj7qmwOfwPRZ9VEsfPnq7KOpNp+mRfraeVuGY0CvCK16+ddQInnbTbOq6tNTPmWtyivj00yDxb3MqXnFa1iHkjG0dGANm3uuydAayC0vZ33HdqyprostkVmlcXnKwW+WJnVtZhq4nEZVo8oy7n4cYvWI8TRNl0a98qKrrdOYVl1O3jt6pFTogmRloUS670/KA2hp593YRtp4ELEaq0pQ79cZxsn3wx3rFUs9t5yL1VimnkIac+fdMGlB5o3HgJdXvquqVz7w8ID1jKSLlg48lPgrpU8yB+xegHqh6qoujT6a8ilfSCN10e75uckT9ChRr/Apm3olLbBXzLqbbdZDFYCpyTEAwBvp70Xpd0YvA5/Me76iu3+yi4cYd9L/VH4I4AsWeDOOAcCDlT8e8zLwybz/K2p/SitDehlPcJNjAF+wwAO9/MIxYLD9nOVM+W+dQCe/N/wWhU/qA1/R9LXKn+A+kS2wF/6hup84AvBGL3/9/ivHAAAAAOBWXv7+4zeOAQAAAMCt8ENBAAAAwO3wvwgDAAAAt8MxAAAAALgdjgFfzFP/Lojmb0h6wiinkvd/Buaz/sUXnt7Vv7Pi8t+acvCve+fv0wAAALHtGND+i32Cv/TQ/67rs3upa70OeuoxwKz+gca3u3AMkLfQy9lTl/csPzidXCubgk0t/7OXzZHA/8raYbt/anNf4noZAACg0h8DyqbB9hDjJuxWru04n7pPvc8xYOWpy3tB2pmfODP7Fr9aB5tR/VnLN/qY+RRwdKy8VJ9lrQAAwOexPAYI9hDXdpxP3adyDHjq8l5wNp/cvGnvJ4N0czwnGHsup07mq1AAAADRMaC7k0rZgY1Fat/8ieaRgOMe9MjecdpGA+n4OlwwljTZaqftN/3urbPba5rGLtvLbaqO04DhavivW8c85WJ1DJiOZYKqIPmVpkvVvhlls/NQgqdspsnvvlFF6q53vRyyfbm09nJmS273c5M+YJ6H39+dl0m1O0sEAABuaP8YYFuTejNx8I8Yxx1P3t5okFXA68eAYQ9UOkr4kkadlY+1Db1VmXFNjlj1Wk05lpPyaGVSYhUw92gSsI67UxbjIxBB8lIqEbqAuTRJftc0DXH2ocRTXs0rJz5ZQ7vewp5KJsf1ciVn4r/63UqX0pFHKc4uFwAAuInwGFD2LOmiqVrvZmr1xqUOEgQcN3/W1AsL2mfr3Vx5i0oJ6GPlDOsqM67JEdNe19ZwnEWV/DKg95mtxu6Uhbep1+FM8iVgkPwRYxom347WrRZM+doaesuchxUP8j45k5ol8/o6bzBJdT2vWpc2AACAOfRfA8Y939h4qm6WtzHleh5w3F1NNzedEjx1tq2RXpbdj9bXfBe4M1bJysvHTHtdW8OxTckwWkNrNFuN3SmLsc1u8lpfswzbNmIcKzCmYcawsWDK19bQW7bFg7zP7BggylherqQqzcfL4bxqYzMAAAARHgN0W5G2HuvdkpcX6i2IxNi2UOuABzc3ndRJA+rPp3z/8WqXOUO9SrGtcQm4O1bJysvHTHtdW8OxzZb8OmC6mK/GkeUd28TJa2WKbVUlYN3GjGMFxjTMGDYWTPnaGnrTFEdue+GYEt7LrTKol7OSjZeTYF61aV8AAIDoGJD3SGW/Ue2WfEc0383Uckv5ZWsfBOw2N+NeZyqdVCTX9FPq2iVde/Yauo5gQ8nF7kZqzPOIaa/xZpmyl2fKCawuWoZBwHA1vJv3me0dxzZ7Y80DBskfsXr0YzKxYMp785qvoTe9JC/JPEjKZ1I7vX/kUYpxjgAAAGJ5DNAtRbXzSEXdHsl1t72L+eYv9fZbSRAwJyLS5dB3ZC1fRQrihTqITkXH8mxsF5jnrCESra2KZ3euZtXL755cQ2+nE0kBqsirgPursZ6yGNuIeKzp8npxkfyuaRrCQ70hjvavM5TCmTXUbiWN6s4R9RAjG7N86Io64SKeV+GNziQJAADuoD8GbNLuzVsltusxq33MVAq82NxkdUDf3+T7Whg2N6MUzUfxAFqqikkd8MhGql2Wo9upVa8La1ivhgQqyZtVwNVqxFNusnYeRKzGqicr9+sM4+RX4jREPWK9vFO7T/nsGlrttWOASN3mvdIozUyFDSQdvJwdenuHNgAAAGY7BviNJ8h7m3O7JeAfyc8Ph7fmvpG/9PF5S18AAPDP9vRjQP4hCDYigLM/o5dPhZfX7I/zj7Qc8dEDAACBJx4D0p95CjYiQC//pM+zPhqrHyUCAAAw7/FDQQAAAAA+FY4BAAAAwO1wDAAAAABuh2MAAAAAcDscAwAAAIDb4RgAAAAA3M7LLxwDAAAAgJt5+ev3XzkGAAAAALfy8vcfv3EMAAAAAG6FHwoCAAAAbof/RRgAAAC4HY4BAAAAwO1wDAAAAABuh2MAAAAAcDscAwAAAIDb4RgAAAAA3A7HAAAAAOB2OAYAAAAAt8MxAAAAALgdjgEAAADA7XAMAAAAAG6HYwAAAABwOxwDAAAAgNvhGAAAAADcDscAAAAA4HY4BgAAAAC3wzEAAAAAuB2OAQAAAMDtcAwAAAAAbodjAAAAAHA7HAMAAACA2+EYAAAAANwOxwAAAADgdjgGAAAAALfz8gvHAAAAAOBmXv76/VeOAQAAAMCtvPz9x28cAwAAAIBb4YeCAAAAgJv56af/Af1pgdRm+b5VAAAAAElFTkSuQmCC)
```

Comment:
Is this the link to binary ? or the binary itself ?

Reply:
This is base64 of the file

Reply:
Replaced above base64 witnet with github image from main branch